buildozer/def.bzl

```diff
@@ -80,6 +83,14 @@ _buildozer = rule(
         "error_on_no_changes": attr.bool(
             doc = "Exit with 3 on success, when no changes were made",
         ),
+        "format_on_write": attr.bool(
+            default = False,
+            doc = "Set to True for format on write using the -buildifier flag. If the `buildifier` attribute is empty, use the built-in formatter",
+        ),
+        "buildifier": attr.label(
+            executable = True,
```

Comment:
I'd like to default to `@com_github_bazelbuild_buildtools` here, but am unsure whether or not your users would want that.